lib/plugins.js

```diff
@@ -181,6 +181,10 @@ exports.categories = {
             url: 'https://github.com/fhemberger/good-logstash',
             description: 'Logstash TCP/UDP broadcasting for Good process monitor'
         },
+        'good-kinesis-reporter': {
+            url: 'https://github.com/vvondra/good-kinesis-repoter',
```

Comment:
There's a small typo here— `repoter` should be `reporter`.

Reply:
oops, thanks for noticing!